src/opnsense/mvc/app/views/OPNsense/Core/firmware.volt

```diff
@@ -435,7 +435,7 @@
                     '<td>' + bold_on + row['flatsize'] + bold_off + '</td>' +
                     '<td>' + bold_on + row['comment'] + bold_off + '</td>' +
                     '<td><button class="btn btn-default btn-xs act_details" data-package="' + row['name'] + '" ' +
-                        ' data-toggle="tooltip" title="More about ' + row['name'] + '">' +
+                        ' data-toggle="tooltip" title="{{ lang._('More about ') }}' + row['name'] + '">' +
```

Comment:
String should be "More about %s" although lang._() will not allow you to inject proper Javascript in this instance ;)

we should probably simplify the labels to "Info", "Remove" and, "Install" (see below)

Reply:
ok, now change these to simple labels to get rid of potential formatting issues:

```
title="{{ lang._('Info) }}"
```

We don't need row['name'] here, my mistake. Thanks!